src/components/TermsOfUse/Terms.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useNavigate } from 'react-router';
+import termsofuseLogo from '../../images/termsofuse-logo.png';
+import { MdOutlineArrowForwardIos } from 'react-icons/md';
+import { MdFavoriteBorder } from 'react-icons/md';
+import FooterForRestoran from '../Restorans/FooterForRestoran/FooterForRestoran';
+const TermsOfUse = () => {
```

Comment:
You again have the different file with same component name? Why did you implement this?